addon/initializers/m3-store.js

```diff
@@ -56,6 +59,18 @@ export function extendStore(Store) {
       return this._super(modelName, id, clientId, storeWrapper);
     },
 
+    /**
+     * A thin wrapper around the API response that knows how to look up relationships.
+     *
+     * @param {string} url The URL path to query.
+     * @param {Object} options
+     * @param {string} [options.method=GET] The HTTP method to use.
+     * @param {Object} [options.params] The parameters to include
+     * @param {string} [options.cacheKey] A string to uniquely identify this request.
+     * @param {boolean} [options.reload=false]
+     * @param {boolean} [options.backgroundReload=false]
```

Comment:
"If true and a cached value exists, issue a non-blocking request but immediately fulfill with the cached value"